### PR TITLE
add branding colour variables to highlights

### PR DESCRIPTION
**Pull request by jwong-d2l**

### CODE REVIEW

**Comment by jwong-d2l** [comment]
I talked with Jeff G, and updated the PR to allow consumers of the navigation to set the color, if they want to.

File: d2l-navigation-highlight-styles.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
 					text-decoration: none;
 				}
 				--d2l-navigation-highlight-base-hover-focus: {
-					color: var(--d2l-color-celestine);
+					color: var(--d2l-navigation-primary-color, var(--d2l-color-celestine));
 					--d2l-icon-fill-color: var(--d2l-color-celestine);
```
Comment (svanherk) [inline d2l-navigation-highlight-styles.html]: Do you also want the icon colour to match?

**Comment by svanherk** [comment]
@jeroach We should add this to the README, while you're there:
```
--d2l-branding-primary-color: used to customize the colour of the top navigation band.
--d2l-navigation-primary-color: used to customize the hover colour of the highlight links and buttons
```

File: d2l-navigation-highlight-styles.html (hunk 1 above)
Comment (svanherk) [reply d2l-navigation-highlight-styles.html]: If there's not a use case right now, we could leave it as blue and expand later?

Comment (jwong-d2l) [reply d2l-navigation-highlight-styles.html]: I think this does make it match the icon.. at least it did in testing.  We'd like it for our immersive ASV.

Comment (svanherk) [reply d2l-navigation-highlight-styles.html]: Oh really?  Are you setting the icon colour with `--d2l-icon-fill-color` yourself?

Comment (jwong-d2l) [reply d2l-navigation-highlight-styles.html]: maybe that's a different icon?  I'll check now

Comment (jwong-d2l) [reply d2l-navigation-highlight-styles.html]: good call, I checked it and it wasn't working anymore.  strange.

**Comment by jwong-d2l** [comment]
revisit after polymer 3!

**Pull request closed without merging**